gravitee-am-management-api/gravitee-am-management-api-service/src/test/java/io/gravitee/am/management/service/impl/upgrades/DefaultIdentityProviderUpgraderTest.java

```diff
@@ -16,6 +16,7 @@
 package io.gravitee.am.management.service.impl.upgrades;
 
 import io.gravitee.am.management.service.IdentityProviderManager;
+import io.gravitee.am.management.service.impl.upgrades.system.upgraders.DefaultIdentityProviderUpgrader;
 import io.gravitee.am.model.IdentityProvider;
 import io.gravitee.am.service.IdentityProviderService;
```

Comment:
[ERROR] Failures: 
[ERROR]   DefaultIdentityProviderUpgraderTest.shouldUpdateDefaultIdp:76 
Wanted but not invoked:
identityProviderService.update(
    <any string>,
    <any string>,
    <custom argument matcher>,
    <any boolean>
);
-> at io.gravitee.am.service.IdentityProviderService.update(IdentityProviderService.java:63)

However, there was exactly 1 interaction with this mock:
identityProviderService.findAll();
-> at io.gravitee.am.management.service.impl.upgrades.system.upgraders.DefaultIdentityProviderUpgrader.upgrade(DefaultIdentityProviderUpgrader.java:55)


[ERROR]   DefaultReporterUpgraderTest.shouldUpdateDefaultIdp:68 
Wanted but not invoked:
reporterService.update(
    <any>,
    <any string>,
    <custom argument matcher>,
    <any boolean>
);
-> at io.gravitee.am.service.ReporterService.update(ReporterService.java:66)

However, there was exactly 1 interaction with this mock:
reporterService.findAll();
-> at io.gravitee.am.management.service.impl.upgrades.system.upgraders.DefaultReporterUpgrader.upgrade(DefaultReporterUpgrader.java:43)


[INFO] 
[ERROR] Tests run: 262, Failures: 2, Errors: 0, Skipped: 0
[INFO]